src/state/trajectory/logics.ts

```diff
@@ -638,6 +640,15 @@ const cancelConversionLogic = createLogic({
     type: CANCEL_CONVERSION,
 });
 
+const setInitialSelectedUIData = createLogic({
+    process(deps: ReduxLogicDeps, dispatch) {
+        const { action } = deps;
+        const uiData = action.payload;
+        dispatch(setSelectedUIDisplayData(uiData));
+    },
+    type: SET_DEFAULT_UI_DATA,
+});
```

Comment:
This logic looks like it just re-dispatches the triggering action with a different type? If not, what am I missing; if so, why not just use that action in place of the one that triggers this logic?

Reply:
Your point is valid!

Context: this was a larger PR, and I got the feedback that I should chunk things up in more bite size pieces. You're correct that this doesn't really need this logic yet, we can just call both `setDefaultUIData` and `setSelectedUIDisplayData` in `ViewerPanel`, and I would happily make that change for now.

I know because I have a crystal ball, that we are likely going to need this asynchronous logic because we won't know what happens first: parsing of the trajectory to derive `defaultUIData` and retrieval from browser storage of `selectedUIData`.

This is a problem of my making! Trying to work backwards from bigger feature branches into smaller PRs can lead to issues like this.

Maybe the defining of the subtasks is the real problem, as this whole PR is somewhat contrived and serves no purpose except to lay the groundwork for forthcoming code, so I wonder what is the greater sin: using this logic to set derived state, or correctly storing state that is never accessed in the first place.